### DISCUSSION TITLE
Allow updating entity_id of scene/automation

### DISCUSSION THREAD

**Discussion by balloob** [home-assistant/frontend#11216] [Other feature requests]
## The request

With https://github.com/home-assistant/home-assistant/pull/31150 scenes/automations that have an ID will become part of the entity registry. This means that we can start manage their entity ID. We need to make this available in the editors.

Also working on this for scripts but it requires some more work.

## The alternatives

Show the entity settings dialog, but that feels a bit weird because now we have 2 places to change the name ?

**Comment by zsarnett** [comment]
Should not be editable in the entity registry but only through the configuration editor for each
